server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/environment/model/UpdateEnvironmentReq.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.collaboration.environment.model;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import lombok.Data;
+
+/**
+ * @Author: Lebie
+ * @Date: 2024/1/12 14:40
+ * @Description: []
+ */
+@Data
+public class UpdateEnvironmentReq {
+    @Size(max = 256, message = "The length of the environment description must be between 0 and 256")
+    private String description;
+
+    @NotNull
+    private EnvironmentStyle style;
```

Comment:
The `EnvironmentStyle` may add more elements for more color style?

Reply:
Yes, added.